src/js/components/InfiniteScroll/__tests__/InfiniteScroll-test.js

```diff
@@ -75,4 +76,52 @@ describe('InfiniteScroll', () => {
     );
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  test('length when step < array', () => {
+    const allItems = Array(1000)
+      .fill()
+      .map((_, i) => `item ${i + 1}`);
+    const step = 50;
+    const { container } = render(
+      <Grommet>
+        <InfiniteScroll
+          items={allItems}
+          // show={117}
+          step={step}
+        >
+          {item => <Box key={item}>{item}</Box>}
+        </InfiniteScroll>
+      </Grommet>,
+    );
+    expect(container.firstChild.children.length).toEqual(step + 1);
```

Comment:
This is a style suggestion which I think helps readability and intent for those who follow afterward. It's a bit more verbose, but helps communicate what each of these values represent.

```suggestion
    const renderedItems = container.firstChild.children.length;
    const expectedItems = step + 1 // comment explaining why + 1
    expect(renderedItems).toEqual(expectedItems);
```

Reply:
Updated tests to include this style suggestion